### PR TITLE
Issue/670 docs

### LINKED ISSUE #670: Fix Docs Build in CI Workflow
**Describe the bug**
Docs cannot be built in the latest release

**To Reproduce**
1. Create a new release

**Expected behavior**
Docs built successfully

**Screenshots**
N/A

**Desktop (please complete the following information):**
N/A

**Additional context**

### PR DESCRIPTION
**Pull request by dennypradipta**
# Monika Pull Request (PR)  

## What feature/issue does this PR add  
This PR fixes #670 

## How did you implement / how did you fix it  
1. Revert back to getStaticProps
2. Run the cron to publish the Docs every 12 AM and 12 PM

## How to test  
1. Run the docs
2. Go to localhost:3000/docs

## Screenshots
image

### CODE REVIEW

**Review by sapiderman — APPROVED** [review]
lgtm
:+1: :v:

File: .github/workflows/docs.yml [hunk 1]
```diff
@@ -4,6 +4,8 @@ on:
   workflow_dispatch:
   release:
     types: [created]
+  schedule:
+    - cron: "0 */12 * * *" # Every 12 AM/PM every day
```
Comment (haricnugraha) [inline .github/workflows/docs.yml:8]: If the article publish once a week, why don't we set the schedule to run once a week. I suggest something like this `0 0 * * 1` or similar.

**Comment by dennypradipta** [comment]
Unlinked #674 because I misunderstand the task

File: .github/workflows/docs.yml (hunk 1 above)
Comment (dennypradipta) [reply .github/workflows/docs.yml]: Changed it to every 9AM on every Wednesday

**Comment by dennypradipta** [comment]
@nicnocquee For the docs, I temporarily disable eslint error in `npm run build` [https://nextjs.org/docs/api-reference/next.config.js/ignoring-eslint](https://nextjs.org/docs/api-reference/next.config.js/ignoring-eslint)

I think if we want to resolve all types and lining errors, it should be in a different ticket because the issue is pretty huge. In the meantime, we want to be able to deploy the docs ASAP

image

**Review by haricnugraha — APPROVED** [review]

**Pull request merged**